python/mxnet/gluon/nn/conv_layers.py

```diff
@@ -716,7 +718,8 @@ class MaxPool1D(_Pooling):
         If padding is non-zero, then the input is implicitly
         zero-padded on both sides for padding number of points.
     layout : str, default 'NCW'
-        Dimension ordering of data and weight. Only supports 'NCW' layout for now.
+        Dimension ordering of data and weight. Only supports 'NCW' and 'NWC'
+        (only with cuDNN) layouts for now.
```

Comment:
it's a bit ambiguous whether it's the class that requires cudnn, or it's the second layout.

Reply:
Glad you pointed out this ambiguity.  I expanded the scope of this work mid-PR and provided cpu and cuda implementations of the 'transposed' layout pooling, making these descriptions incorrect.  I've changed the text now to be simpler and correct.  Ditto for Symbol's Pooling description.